docs/UI/Alert/index.md

```diff
@@ -46,6 +46,7 @@ An alert may be sent to the HMI for an application which is not currently active
 |progressIndicator|Boolean|false||
 |alertType|[Common.AlertType](../../common/enums/#alerttype)|true||
 |appID|Integer|true||
+|cancelID|Integer|false|can be used later with CancelInteraction RPC to specifically close this alert|
```

Comment:
This table does not have a description section, you included this in the `Additional` section, which includes the bounds for this parameter.

Reply:
fixed for all 4 interaction types in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0